packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -293,9 +293,7 @@
   "Avatar_url_invalid_or_error": "The url provided is invalid or not accessible. Please try again, but with a different url.",
   "away": "away",
   "Away": "Away",
-  "away_female": "away",
```

Comment:
Was this key never used anywhere?

Reply:
Searched for it on the Rocket.Chat repository and couldn't find any location apart from the JSON files

Reply:
I don't think we have used gender for i18n so far

Reply:
Yes, I couldn't find it being used anywhere else but there were some keys there that differentiated gender (mostly status keys)